es-space-mgmt/main/default/lwc/customerList/__tests__/customerList.test.js

```diff
@@ -8,22 +8,9 @@ import FLOW_STATUS_CHANGE_MC from '@salesforce/messageChannel/Flow_Status_Change
 
 import { refreshApex } from '@salesforce/apex';
 
-import {
-    registerApexTestWireAdapter,
-    registerTestWireAdapter
-} from '@salesforce/sfdx-lwc-jest';
-
 // Realistic data with a list of customers
 const mockCustomerList = require('./data/getCustomerList.json');
 
-// Register as Apex wire adapter. Some tests verify that data is retrieved.
-const getCustomerListAdapter = registerApexTestWireAdapter(getCustomerList);
```

Comment:
@msrivastav13 the platform does not auto-mock apex methods (see my [previous comment](https://github.com/trailheadapps/easy-spaces-lwc/pull/257#discussion_r661826190)), this module will need to be mocked. One way of doing this is:

```js

jest.mock(
    '@salesforce/apex/reservationManagerController.getCustomerList',
    () => {
        const { createApexTestWireAdapter } = require('@salesforce/sfdx-lwc-jest');
        return {
            default: createApexTestWireAdapter(jest.fn().mockImplementation(() => Promise.resolve())),
        };
    },
    { virtual: true }
);
```

Reply:
@jodarove I simplified it to be like below

`jest.mock(
    '@salesforce/apex/reservationManagerController.getCustomerList',
    () => {
        const { createApexTestWireAdapter } = require('@salesforce/sfdx-lwc-jest');
        return {
            default: createApexTestWireAdapter(jest.fn()),
        };
    },
    { virtual: true }
);`

It worked fine. 

If you do not mind could you review the commits [here](https://github.com/trailheadapps/easy-spaces-lwc/pull/257/commits/e2e8f9397285a2015089f1880692d2e0af344be5)

Reply:
@pozil Will merge once I get approval from you. 

We might want to broadcast this to a larger audience (maybe Twitter or stackexchange or Salesforce Developers Trailblazer Community)as they will be in the same boat if they are upgrading to the newer version.

Also whoever wrote this [module](https://trailhead.salesforce.com/content/learn/modules/test-lightning-web-components/write-a-jest-test-for-wire-service) on the trailhead, needs to be notified I guess.